Makefrag

```diff
@@ -26,17 +27,21 @@ OBJDUMP := $(RISCV_PREFIX)objdump
 OBJCOPY := $(RISCV_PREFIX)objcopy
 EMU ?= $(FLEXPRET_ROOT_DIR)/emulator/fp-emu # Verilator C++ emulator
 
-include $(FLEXPRET_ROOT_DIR)/defines.mk
-
-CFLAGS = -g -static -O0 -march=rv32i -mabi=ilp32 -nostartfiles --specs=nosys.specs $(APP_DEFS) $(DEFINES)
-INCS = $(LIB_INCS) $(APP_INCS)
-LFLAGS = -T $(LINKER_SCRIPT) -L $(FP_LIB_DIR)/linker -Xlinker -Map=$(NAME).map 
+CFLAGS += -g -static -O0 -march=rv32i -mabi=ilp32 -nostartfiles --specs=nosys.specs -Werror=stack-usage=$(STACKSIZE) $(APP_DEFS) $(DEFINES)
```

Comment:
`-Werror-stack-usage` throws a compile-time error if the compiler detects that the variables used in a function are too big for the stack (i.e., the STACKSIZE variable). To get this variable, the swconfig.mk is included on line 19.